### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by sumitrajpal29**
Update game over message so people know how to play again!

### CODE REVIEW

**Review by sumitrajpal29 — COMMENTED** [review]
You cannot approve your own pull requests, So...

**Comment by arunrajpal29** [comment]
Good Work!

**Comment by sumitrajpal29** [comment]
> Good Work!

Thank you.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (sumitrajpal29) [inline index.html:51]: Change something here...

Comment (sumitrajpal29) [inline index.html:51]: Nothing to add here.

**Review by sumitrajpal29 — COMMENTED** [review]
All is ok, go ahead.

**Review by arunrajpal29 — APPROVED** [review]

**Pull request merged**